dbms/src/Storages/Page/tools/PageCtl/PageStorageCtlV2.cpp

```diff
@@ -277,6 +277,7 @@ void dump_all_entries(PageFileSet & page_files, int32_t mode)
                     break;
                 default:
                     throw DB::Exception(DB::ErrorCodes::LOGICAL_ERROR, "illegal type: {}", magic_enum::enum_name(record.type));
+                    break;
```

Comment:
It is not necessary. It is modified accidentaly. I left it here just for consistency, because sometimes we add this extra line: https://github.com/pingcap/tiflash/blob/e51175250dbd97b22bac40a35d89713616da8b87/dbms/src/Storages/Page/V2/PageFile.cpp#L125